test/autoschedulers/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(HalideTestHelpers)
```

Comment:
This shouldn't be necessary as it's already included in the CMakeLists.txt above this one... why is it here?

Reply:
If the answer is "removing it breaks things" then please open an issue and we can land this PR. Otherwise, remove this line.

Reply:
Copy-paste error, removed